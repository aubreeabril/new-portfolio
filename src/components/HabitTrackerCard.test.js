import React from "react";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";
import HabitTrackerCard from "./HabitTrackerCard";

describe("Habit Tracker Card", () => {
  it("renders the habit tracker card", () => {
    const wrapper = shallow(<HabitTrackerCard />);
    expect(wrapper.find(".habit-tracker").length).toEqual(1);
  });

  it("matches its snapshot", () => {
    const section = renderer.create(<HabitTrackerCard />).toJSON();
    expect(section).toMatchSnapshot();
  });
});
