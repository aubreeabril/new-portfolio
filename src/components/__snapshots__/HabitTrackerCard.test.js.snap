// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Habit Tracker Card matches its snapshot 1`] = `
<div
  className="project-card habit-tracker card"
>
  <div
    className="card-image"
  >
    <iframe
      allow="autoplay; encrypted-media"
      allowfullscreen={true}
      frameborder="0"
      src="https://www.youtube.com/embed/KeW_IQzL-3I"
      title="Habit Tracker demo"
      width="100%"
    />
  </div>
  <h5
    className="card-title"
    style={
      Object {
        "padding": "10px",
      }
    }
  >
    Habit Tracker
  </h5>
  <p
    className="project-text card-text"
  >
    An application for finding and saving recipes. Built with a Rails API backend and vanilla JS frontend, styled using Semantic UI.
  </p>
</div>
`;
